build/babel-plugins/babel-plugin-gql/test/fixtures/expected-import-destructuring

```diff
@@ -0,0 +1,9 @@
+import { gql } from 'fusion-apollo';
```

Comment:
I think fusion-apollo needs a named gql export, otherwise this may cause errors.

Ideally this is something like:

```js
export const gql = (() => {
  throw new Error("gql called at runtime");
} /*: string => string */);
```

Then $FlowFixMe can be removed too.

Reply:
Yea, I was going to do that next.

Reply:
The $FlowFixMe is actually there because we are not installing fusion-apollo